src/SignalR/common/Shared/MessageBuffer.cs

```diff
@@ -268,6 +297,8 @@ internal async Task ResendAsync(PipeWriter writer)
         catch (Exception ex)
         {
             tcs.SetException(ex);
+            // Observe exception in case WriteAsync isn't waiting on the task
```

Comment:
When does this happen?

Reply:
I guess this code should never error since we don't complete the reader with an exception.

So now the only reasons it would error is if we got here with an `IHubProtocol` implementation that threw when writing a sequence message, or a custom implementation of a transport.

Reply:
Can we pass an `ILogger` to the `MessageBuffer` for the rare case this does happen and 	WriteAsync doesn't await it?